harvard-lts/CURIOSity
# Hide status

**Hides Devise messages we dont want to see**

Ticket: [#308](https://github.com/harvard-lts/CURIOSity/issues/308)

* * *


# What does this Pull Request do?
This hides the 2 devise messages that due to CAS show AFTER you logged in rather than before like they were supposed to

# How should this be tested?

This is on dev (testing locally with CAS isn't possible). To test the 2 messages:
- open an incognito window
- attempt to go to https://curiosity-dev.lib.harvard.edu/site/edit
- login with key
- you should NOT see a message about needing to log in (you will on qa)
- then leave that window for a little over 2 hours to let your cookie expire -- when you re-auth (either manually or auto if your key login was refreshed recently) you SHOULD NOT see a message about needing to re login (you will on qa)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
